### PR TITLE
allow API token to be retrieved from environment variable #24

### LINKED ISSUE #24: Allow API token to be retrieved from an environment variable
While I was setting up my dev environment over and over as I worked on https://github.com/IQSS/dataverse/pull/7363 I started getting tired of editing dvconfig.py each time to add the new API token.

### PR DESCRIPTION
**Pull request by pdurbin**
Closes #24 - Allow API token to be retrieved from an environment variable

I'm aware that there's a similar solution in pull request #14 

I'm also aware that here the dvconfig.py file is edited directly: https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/56ae75de0a796e515200d6830115f60e6ce3fd5e/tasks/sampledata.yml#L59

This is a small chunk and makes things easier for me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Copy `dvconfig.py.sample` to `dvconfig.py` (see the `cp` command below) and add
     cp dvconfig.py.sample dvconfig.py
     vi dvconfig.py
 
+Note that the environment variable `$API_TOKEN` will override `api_token` in `dvconfig.py`.
```
Comment (scolapasta) [inline README.md:36]: My only question here is which token should take precedence, the environment variable or the one in dvconfig.py? (I have not looked at this code very deeply overall, so not necessarily advocating for the switch)

Comment (djbrooke) [reply README.md:36]: test

Comment (djbrooke) [reply README.md:36]: test

**Comment by djbrooke** [comment]
test

File: README.md (hunk 1 above)
Comment (scolapasta) [reply README.md:36]: I searched online and it seems standard for environment variable to take precedence.

**Review by scolapasta — APPROVED** [review]

**Pull request merged**